js/mexc3.js

```diff
@@ -4002,6 +4025,123 @@ module.exports = class mexc3 extends Exchange {
+        //
+        //     [
+        //         [
+        //             {
+        //                 "symbol": "BTCUSDT",
+        //                 "leverage": "10",
+        //                 "data": [
+        //                     {
+        //                         "coin": "BTC",
+        //                         "hourInterest": "0.00026125",
+        //                         "borrowLimit": "270"
+        //                     }, {
+        //                         "coin": "USDT",
+        //                         "hourInterest": "0.000475",
+        //                         "borrowLimit": "2100000"
+        //                     }
+        //                 ]
+        //             }
+        //         ]
+        //     ]
+        //
+        return this.parseBorrowRates (response, 'coin');
+    }
+
+    parseBorrowInterest (info, market = undefined) {
+        const coin = this.safeString (info, 'asset');
+        const timestamp = this.safeInteger (info, 'timestamp');
+        const datetime = this.iso8601 (timestamp);
+        const remainInterest = this.safeString (info, 'remainInterest');
+        const repayInterest = this.safeString (info, 'repayInterest');
+        const remainAmount = this.safeString (info, 'remainAmount');
+        const repayAmount = this.safeString (info, 'repayAmount');
+        const totalInterest = Precise.stringAdd (remainInterest, repayInterest);
+        const totalAmount = Precise.stringAdd (remainAmount, repayAmount);
+        return {
+            'account': 'isolated',
+            'symbol': this.safeString (info, 'symbol'),
+            'marginMode': 'isolated',
+            'currency': this.safeCurrencyCode (coin),
+            'interest': this.parseNumber (totalInterest),
+            'interestRate': undefined,
+            'amountBorrowed': this.parseNumber (totalAmount),
+            'timestamp': timestamp,
+            'datetime': datetime,
+            'info': info,
+        };
+    }
+
+    async fetchBorrowInterest (code = undefined, symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        if (code === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchBorrowInterest() requires a currency code argument');
+        }
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchBorrowInterest() requires a symbol argument');
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const currency = this.safeCurrency (code);
+        const request = {
+            'asset': currency['code'];
+            'symbol': market['id'],
+        };
+        if (since !== undefined) {
+            request['startTime'] = parseInt (since / 1000);
+        }
+        const response = await this.spotPrivateGetMarginLoan (this.extend (request, params));
+        //
+        //     {
+        //         "total":0,
+        //         "rows":[
+        //         ]
```

Comment:
We should include a few rows here.

Reply:
@kroitor I added example after pass their KYC.